libr/core/cmd_print.c

```diff
@@ -5851,6 +5851,92 @@ l = use_blocksize;
 					                len, R_PRINT_STRING_ZEROEND);
 				}
 			}
+			break;
+		case 't': // "pst"
+			if (l > 0) {
+				ut8 var[len];
```

Comment:
dont use variable length arrays and even less in the stack

Reply:
this is not solved yet

Reply:
@trufae : Done! See https://github.com/radareorg/radare2/pull/16621/commits/f178654dd136581fed7a6416d81d65a21921d725#diff-55abf099b2aa9100745103d790a1ad9cR5664